packages/tpetra/core/src/Tpetra_Details_computeOffsets.hpp

```diff
@@ -294,6 +294,7 @@ computeOffsetsFromCounts (const ExecutionSpace& execSpace,
             countsAPS(Kokkos::ViewAllocateWithoutInitializing("APS_counts"),
                       numCounts);
 
+    // DEEP_COPY REVIEW - NOT TESTED
```

Comment:
The code in this ifdef block was added to help debug / workaround the so-called "UVM bug" that turned out to be a Kokkos::parallel_scan bug.  The ifdef block should be removed.  I will do it.